docs/windows.md

```diff
@@ -328,3 +330,47 @@ the HNS Network created by antrea-agent is removed, and the Open vSwitch
 Extension is disabled by default. In this case, the stale OVS bridge and ports
 should be removed. A help script [Clean-AntreaNetwork.ps1](https://raw.githubusercontent.com/vmware-tanzu/antrea/master/hack/windows/Clean-AntreaNetwork.ps1)
 can be used to clean the OVS bridge.
+
+2. Hyper-V feature cannot be installed on Windows node due to the processor does
+not have required virtualization capabilities.
+
+If the processor of the Windows node does not have required virtualization
```

Comment:
You may need to indent the following paragraphs, otherwise they don't show as part of the 2nd issue.

Reply:
Done.